packages/calling-component-bindings/review/calling-component-bindings.api.md

```diff
@@ -194,6 +196,9 @@ export const getDeviceManager: (state: CallClientState) => DeviceManagerState;
 // @public (undocumented)
 export const getDisplayName: (state: CallClientState) => string | undefined;
 
+// @public (undocumented)
+export const getDominantSpeakers: (state: CallClientState, props: CallingBaseSelectorProps) => undefined | DominantSpeakersInfo;
```

Comment:
This shouldn't be part of the public API?

This isn't used by any of our components directly, so it shouldn't be exposed.

Actually, I think there are other base selectors that need to be hidden.

Reply:
So currently, everything inside baseSelector is being exported through index file.
Can create a bug for this cleanup.